e2e/package.json

```diff
@@ -20,28 +20,28 @@
   "dependencies": {
     "@frequency-chain/api-augment": "file:../js/api-augment/dist/frequency-chain-api-augment-0.0.0.tgz",
     "@helia/unixfs": "^3.0.7",
-    "@noble/curves": "^1.5.0",
-    "@polkadot/api": "12.3.1",
-    "@polkadot/types": "12.3.1",
-    "@polkadot/util": "13.0.2",
-    "helia": "^4.2.5",
-    "multiformats": "^13.2.2",
+    "@noble/curves": "^1.6.0",
+    "@polkadot/api": "13.0.1",
+    "@polkadot/types": "13.0.1",
+    "@polkadot/util": "13.1.1",
+    "helia": "^4.2.6",
+    "multiformats": "^13.3.0",
```

Comment:
<rant>
Why can't Polkadot version all of their packages the same?
</rant>

Reply:
It isn't quite as bad as it feels. The utils and crypto-utils are versioned differently, and the rest are the same.

Why? Not sure.